index.html

```diff
@@ -28,6 +28,111 @@ <h1>I'm Ahmed <br> Glad to see <br> you!</h1>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <a href="#"><h4>See Project</h4></a>
+    </div>
+    <div class="card2">
+      <img src="works/Snapshoot-Portfolio2.svg" alt="snaphot" class="p-img">
+      <h2>Multi-Post <br>Stories</h2>
+      <section class="u-c">
+        <h5>CANOPY</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>Back End Dev</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>2015</h5>  
+      </section>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="u-html">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <a href="#"><h4>See Project</h4></a>
+    </div>
+    <div class="card3">
+      <img src="works/Snapshoot-Portfolio3.svg" alt="snaphot" class="p-img">
+      <h2>Tonic</h2>
+      <section class="u-c">
+        <h5>CANOPY</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>Back End Dev</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>2015</h5>  
+      </section>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="u-html">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <a href="#"><h4>See Project</h4></a>
+    </div>
+    <div class="card4">
+      <img src="works/Snapshoot-Portfolio4.svg" alt="snaphot" class="p-img">
+      <h2>Multi-Post <br>Stories</h2>
+      <section class="u-c">
+        <h5>CANOPY</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>Back End Dev</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>2015</h5>  
+      </section>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="u-html">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <a href="#"><h4>See Project</h4></a>
+    </div>      
+  </section>
+  <section class="about-section">
+    <h1>About<br>Myself</h1>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <a href="#" class="connect">Let's Connect</a>
+    <ul class="socialmeadia-container">
+      <li class="icons"><a href="#"><img src="./svgs/Vector.svg" alt="svg"></a></li>
+      <li class="icons"><a href="#"><img src="./svgs/Linkedinicon.svg" alt="svg"></a></li>
+      <li class="icons"><a href="#"><img src="./svgs/Vector(4).svg" alt="svg"></a></li>
+      <li class="icons"><a href="#"><img src="./svgs/Vector(2).svg" alt="svg"></a></li>
+      <li class="icons"><a href="#"><img src="./svgs/Vector(3).svg" alt="svg"></a></li>
+    </ul>
+    <ul class="languages">
```

Comment:
- [ ] there is a missing 'Get My Resume' Button before the languages list , kindly add it with the styling effects of a button like I mentioned before : 
the missing button : 
![mbtn](https://user-images.githubusercontent.com/89282221/185711063-f2aec8d1-0814-4798-a903-275b545b3745.png)